app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java

```diff
@@ -0,0 +1,74 @@
+package com.pcsa.views.reporting;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import com.pcsa.beans.reporting.LocationDetails;
+import com.pcsabuddhi.R;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Buddhiprabha Erabadda
+ *
+ * Allows the user to call Post Staff in case of crime. The details for the
+ * current location will be set by changing the location
+ */
+public class ContactPostStaff extends Activity implements AdapterView.OnItemSelectedListener {
+
+    Button contact_pcmo;
```

Comment:
Naming should be done, showing the purpose of the element. Also, java coding standards prefer usage of camel case for variables